controller/ovn-controller.c

```diff
@@ -2720,6 +2734,62 @@ main(int argc, char *argv[])
     OVS_NODES
 #undef OVS_NODE
 
+    /*** Begin DDlog test code ***/
+    // DDlog vars
+    ddlog_prog prog;
+    table_id input_table;
+    table_id output_table;
+    const char *input_relation = "OVN_Southbound::Port_Group";
+    const char *output_relation = "PortGroupSubset";
+
+    prog = ddlog_run(1, true, NULL, NULL);
+    if (!prog) {
+        ovs_fatal(0, "DDlog instance could not be created");
+    }
+
+    input_table = ddlog_get_table_id(prog, input_relation);
+    if (input_table == -1)
```

Comment:
I'm sure it doesn't matter for this test code, but OVS/OVN style calls for always putting `{}` around the statements gated by an `if`.